### PR TITLE
Feature Behavioral Patterns

### PR DESCRIPTION
**Pull request by lharzenetter**
- [x] Ensure that you followed http://eclipse.github.io/winery/dev/ToolChain#github---prepare-pull-request. Especially, we require **a single commit**
- [ ] Ensure that the commit message is [a good commit message](https://github.com/joelparkerhenderson/git_commit_message)
- [x] Ensure to use auto format in **all** files
- [x] Ensure that you appear in `NOTICE` at Copyright Holders
- [x] Tests created for changes
- [ ] Documentation updated (if needed)
- [ ] Screenshots added (for UI changes)

### CODE REVIEW

File: org.eclipse.winery.frontends/app/topologymodeler/src/app/canvas/canvas.component.ts [hunk 1]
```diff
@@ -1155,13 +1156,32 @@ export class CanvasComponent implements OnInit, OnDestroy, OnChanges, AfterViewI
...
+                const list: TPolicy[] = newRelationship.policies.policy;
+                labelString += '<br>';
+                for (const value of list) {
+                    const visual = this.entityTypes.policyTypeVisuals.find(
+                        policyTypeVisual => policyTypeVisual.typeId === value.policyType
+                    );
+
+                    if (visual && visual.imageUrl) {
+                        labelString += '<img style="display: block; margin-left: auto; margin-right: auto; margin-top: 5px;' +
+                            ' max-width: 40px; max-height: 40px;" src="' + visual.imageUrl + '" />';
```
Comment (miwurster) [inline org.eclipse.winery.frontends/app/topologymodeler/src/app/canvas/canvas.component.ts]: Be aware or XSS attacks

File: org.eclipse.winery.frontends/app/topologymodeler/src/app/node/node.component.html [hunk 2]
```diff
@@ -30,12 +30,13 @@
         {{ nodeTemplate.state }}
     </div>
 
-    <div *ngIf="policyIcons">
+    <div *ngIf="policyIcons" style="position: absolute; top: -27px">
```
Comment (miwurster) [inline org.eclipse.winery.frontends/app/topologymodeler/src/app/node/node.component.html]: Avoid using inline CSS

Comment (lharzenetter) [reply org.eclipse.winery.frontends/app/topologymodeler/src/app/node/node.component.html]: I tried to use a CSS class, but somehow it did not get applied even though it was shown in the inspector...

Comment (miwurster) [reply org.eclipse.winery.frontends/app/topologymodeler/src/app/node/node.component.html]: Maybe try a more specific selector?!

**Pull request merged**